docs/local.md

```diff
@@ -39,19 +39,28 @@
 
  - [install **golangci-lint**](https://golangci-lint.run/usage/install/#local-installation) for code quality checks
 
+ ---
```

Comment:
@jkremser  I think this disclaimer shouldn't be a part of "Environment prerequisites" section, but predate it, e.g.
![image](https://user-images.githubusercontent.com/502695/144855480-d7442c2c-9889-43dd-9b9c-fb5ee7424bab.png)
Also it is probably worth to mention that by running `make` or `make help` it is possible to get the list of user-centric make targets, e.g.:
```
> make         
check                     Check project integrity
demo                      Execute end-to-end demo
deploy-full-local-setup   Deploy full local multicluster setup (k3d >= 5.1.0)
deploy-gslb-operator      Deploy k8gb operator
deploy-test-apps          Deploy Podinfo (example app) and Apply Gslb Custom Resources
deploy-test-version       Upgrade k8gb to the test version on existing clusters
destroy-full-local-setup  Destroy full local multicluster setup
dns-tools                 Run temporary dnstools pod for debugging DNS issues
help                      Show this help
```

Reply:
right, this looks better, lemme change it. I wouldn't put the whole output of `make help` there though, the tutorial is pretty long already.

perhaps one more sentence in that note block saying: "For more user-centric targets in that makefile consult `make help`" (?)

Reply:
@jkremser Agree 👍 , let's keep it simple